### ISSUE TITLE
Delete local image if not in use anymore

### ISSUE THREAD

**Issue by BotoX** [flagrow/flarum-ext-image-upload#53]
Delete locally uploaded images if their parent post (or the entire thread) is deleted (forever).
Editing the post and removing the image would also leave the image on the server, wasting disk space.

**Comment by Luceos** [comment]
Yes that was the whole idea of storing them to the database as well. Feel free to PR the necessary changes, because right now Flagrow is in hibernation until further notice.
